### PR TITLE
[Browser Isolation] Added network dependencies

### PR DESCRIPTION
**Pull request by obezuk**
Network dependencies to enable connecting to Browser Isolation service in firewalled environments

### CODE REVIEW

File: content/cloudflare-one/policies/browser-isolation/network-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+pcx-content-type: reference
+title: Browser Isolation with firewall
+weight: 9
+---
+
+# Browser Isolation with firewall
+
+If your organization uses a firewall or other policies to restrict Internet traffic, you may need to make a few changes to allow Browser Isolation to connect.
+
+## Remoting client
```
Comment (ranbel) [inline content/cloudflare-one/policies/browser-isolation/network-dependencies.md:11]: is Remoting client different from the WARP client?

Comment (obezuk) [reply content/cloudflare-one/policies/browser-isolation/network-dependencies.md:11]: Yes, it's the web based client we serve to the local web browser that connects to the remote browser.

File: content/cloudflare-one/policies/browser-isolation/network-dependencies.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+pcx-content-type: reference
+title: Browser Isolation with firewall
+weight: 9
+---
+
+# Browser Isolation with firewall
+
+If your organization uses a firewall or other policies to restrict Internet traffic, you may need to make a few changes to allow Browser Isolation to connect.
```
Comment (ranbel) [inline content/cloudflare-one/policies/browser-isolation/network-dependencies.md:9]: "may" is a bit vague. How do users know which steps they need to take? Should we recommend that they configure all of the permissions listed below?

**Comment by ranbel** [comment]
Page is modeled after https://developers.cloudflare.com/cloudflare-one/connections/connect-devices/warp/deployment/firewall

File: content/cloudflare-one/policies/browser-isolation/network-dependencies.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Remoting client
+
+Isolated pages are served the remoting client. This client communicates to Cloudflare's network via HTTPS and WebRTC.
```
Comment (ranbel) [inline content/cloudflare-one/policies/browser-isolation/network-dependencies.md]:
```suggestion
Isolated pages are served by the remoting client. This client communicates to Cloudflare's network via HTTPS and WebRTC.
```

File: content/cloudflare-one/policies/browser-isolation/network-dependencies.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+### Remoting Client (Services)
+
+Provides static assets and API endpoints required for Browser Isolation to function.
```
Comment (ranbel) [inline content/cloudflare-one/policies/browser-isolation/network-dependencies.md]:
```suggestion
The remoting client provides static assets and API endpoints. For Browser Isolation to function, you must allow:
```

File: content/cloudflare-one/policies/browser-isolation/network-dependencies.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+Users connecting through Clientless Web Isolation also require connectivity to Cloudflare Access:
+
+- Allow HTTPS traffic to `https://<auth_domain>.cloudflareaccess.com` on port 443
+
+### WebRTC channel
+
+Browse Isolation uses WebRTC for low-latency communication between the local browser and the remote browser. 
+
+These are the IP addresses that the Remoting Client will connect to. All WebRTC traffic from your device to the remote browser will go through these IP addresses.
```
Comment (ranbel) [inline content/cloudflare-one/policies/browser-isolation/network-dependencies.md]:
```suggestion
In order to pass WebRTC traffic, the remoting client must be able to connect to the following IP addresses:
```

File: content/cloudflare-one/policies/browser-isolation/network-dependencies.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+#### Clientless Web Isolation
+
+Users connecting through Clientless Web Isolation also require connectivity to Cloudflare Access:
+
+- Allow HTTPS traffic to `https://<auth_domain>.cloudflareaccess.com` on port 443
+
+### WebRTC channel
+
+Browse Isolation uses WebRTC for low-latency communication between the local browser and the remote browser.
```
Comment (ranbel) [inline content/cloudflare-one/policies/browser-isolation/network-dependencies.md]:
```suggestion
Browser Isolation uses WebRTC for low-latency communication between the local browser and the remote browser. 
```

File: content/cloudflare-one/policies/browser-isolation/network-dependencies.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+## Remoting client
+
+Isolated pages are served the remoting client. This client communicates to Cloudflare's network via HTTPS and WebRTC.
+
+### Remoting Client (Services)
+
+Provides static assets and API endpoints required for Browser Isolation to function.
+
+- Allow HTTPS traffic to `*.browser.run` on port 443
```
Comment (ranbel) [inline content/cloudflare-one/policies/browser-isolation/network-dependencies.md]:
```suggestion
- HTTPS traffic to `*.browser.run` on port 443
```

File: content/cloudflare-one/policies/browser-isolation/network-dependencies.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+#### Clientless Web Isolation
+
+Users connecting through Clientless Web Isolation also require connectivity to Cloudflare Access:
+
+- Allow HTTPS traffic to `https://<auth_domain>.cloudflareaccess.com` on port 443
```
Comment (ranbel) [inline content/cloudflare-one/policies/browser-isolation/network-dependencies.md]:
```suggestion
- HTTPS traffic to `https://<auth_domain>.cloudflareaccess.com` on port 443
```

File: content/cloudflare-one/policies/browser-isolation/network-dependencies.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+### Remoting Client (Services)
+
+Provides static assets and API endpoints required for Browser Isolation to function.
+
+- Allow HTTPS traffic to `*.browser.run` on port 443
+
+#### Clientless Web Isolation
+
+Users connecting through Clientless Web Isolation also require connectivity to Cloudflare Access:
```
Comment (ranbel) [inline content/cloudflare-one/policies/browser-isolation/network-dependencies.md]:
```suggestion
Users connecting through Clientless Web Isolation also require connectivity to Cloudflare Access. For users to connect to Access, you must allow:
```

File: content/cloudflare-one/policies/browser-isolation/network-dependencies.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+## Remoting client
+
+Isolated pages are served by the remoting client. This client communicates to Cloudflare's network via HTTPS and WebRTC.
+
+### Remoting Client (Services)
+
+The remoting client provides static assets and API endpoints. For Browser Isolation to function, you must allow:
+
+- HTTPS traffic to `*.browser.run` on port 443
```
Comment (ranbel) [inline content/cloudflare-one/policies/browser-isolation/network-dependencies.md]:
```suggestion
- HTTPS traffic to `*.browser.run` on port `443`
```

File: content/cloudflare-one/policies/browser-isolation/network-dependencies.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+#### Clientless Web Isolation
+
+Users connecting through Clientless Web Isolation also require connectivity to Cloudflare Access. For users to connect to Access, you must allow:
+
+- HTTPS traffic to `https://<auth_domain>.cloudflareaccess.com` on port 443
```
Comment (ranbel) [inline content/cloudflare-one/policies/browser-isolation/network-dependencies.md]:
```suggestion
- HTTPS traffic to `https://<team-name>.cloudflareaccess.com` on port `443`
```

**Review by ranbel — APPROVED** [review]

**Comment by obezuk** [comment]
Thanks!

**Pull request merged**